backend/dal/Migrations/20210114232914_v01.06.01.Designer.cs

```diff
@@ -0,0 +1,3412 @@
+﻿// <auto-generated />
+using System;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Infrastructure;
+using Microsoft.EntityFrameworkCore.Metadata;
+using Microsoft.EntityFrameworkCore.Migrations;
+using Microsoft.EntityFrameworkCore.Storage.ValueConversion;
+using NetTopologySuite.Geometries;
+using Pims.Dal;
+
+namespace Pims.Dal.Migrations
+{
+    [DbContext(typeof(PimsContext))]
+    [Migration("20210114232914_v01.06.01")]
+    partial class v010601
+    {
+        protected override void BuildTargetModel(ModelBuilder modelBuilder)
```

Comment:
This doesn't look right, looks like you're migration is for some reason trying to do something with Access Requests.

Reply:
I just checked this against other .Designer files from other database changes and they're all the same, isn't this just listing everything in alphabetical order?

Reply:
You're right they all have this stuff.  hmmm, never noticed this before.